front-end/legeno-around-here/src/components/api/API.js

```diff
@@ -556,14 +559,19 @@ export const findOthersProfileById = async ({ accessToken, userId, history }) =>
       const userResponse = await response.data;
       return userResponse;
     })
-    .catch((error) => {
+    .catch(async (error) => {
+      await redirectLoginWhenUnauthorized(error, history);
+      if (error.response && error.response.status === 404) {
+        alert("존재하지 않는 회원입니다.");
+        window.history.back();
```

Comment:
```suggestion
        history.goBack();
```
react-router-dom을 사용해서 다음과 같이 수정하시면 좋을 것 같습니다 :)

Reply:
수정했습니다!!